picard/formats/__init__.py

```diff
@@ -84,107 +84,6 @@ def open(filename):
         return None
 
 
-def _insert_bytes_no_mmap(fobj, size, offset, BUFFER_SIZE=2**16):
```

Comment:
https://tickets.metabrainz.org/browse/PICARD-933

Removed it as per the discussion. Can undo it if required :)